app/models/responsible_body.rb

```diff
@@ -88,4 +88,35 @@ def self.with_completed_preorder_info_count
       ",
     )
   end
+
+  def is_ordering_for_schools?
+    schools.that_are_centrally_managed.count.positive?
+  end
+
+  def has_centrally_managed_schools_that_can_order_now?
+    schools.that_are_centrally_managed.that_can_order_now.count.positive?
+  end
+
+  def has_schools_that_can_order_devices_now?
+    schools.that_will_order_devices.that_can_order_now.count.positive?
+  end
+
+  def has_any_schools_that_can_order_now?
+    schools.that_can_order_now.count.positive?
+  end
+
+  def count_of_schools_we_order_for_link_text
```

Comment:
Not sure about this method being on the model. Actually keeping the logic in the view allows you to use `https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize` which makes things a bit more readable:
```
    <p class="govuk-body">None of the
      <%= govuk_link_to '', responsible_body_devices_schools_path do %>
        <%= pluralize(@responsible_body.schools.that_are_centrally_managed.count, 'school') %> you will be placing orders for
      <% end %>
      have local coronavirus restrictions.
    </p>
```

Reply:
Think that one should also be handling 'has/have' to in the pluralisation and once it's more that one item it gets a bit more messy in the view - I agree these should't be on the model, would moving it into the `view_helper` be a better idea?

Reply:
> would moving it into the view_helper be a better idea?

I'm fine with that.

Reply:
Actually most of these were reasonable to do inline with just one going in the view helper